e2e-tests/ios-only/financial-connections-session.yml

```diff
@@ -9,16 +9,22 @@ appId: ${APP_ID}
     text: "Collect session"
     retryTapIfNoChange: false
 - assertVisible:
-    text: "Manually verify instead"
-- tapOn: "Manually verify instead"
```

Comment:
Seems like Maestro is unable to hit this on iOS. We switch to longer flow of actually selecting an institution and account.